CMakeLists.txt

```diff
@@ -12,12 +12,13 @@ if(NOT WIN32)
       ncursesw5-config --terminfo-dirs 2>/dev/null || \
       ncurses5-config  --terminfo-dirs 2>/dev/null || \
       echo '/etc/terminfo:/lib/terminfo:/usr/share/terminfo:/usr/lib/terminfo:/usr/local/share/terminfo:/usr/local/lib/terminfo'"
-    OUTPUT_VARIABLE TERMINFO_DIRS)
+    OUTPUT_VARIABLE TERMINFO_DIRS_DEFAULT)
   # Remove trailing newline
-  string(STRIP "${TERMINFO_DIRS}" TERMINFO_DIRS)
+  string(STRIP "${TERMINFO_DIRS_DEFAULT}" TERMINFO_DIRS_DEFAULT)
 else()
-  set(TERMINFO_DIRS "")
+  set(TERMINFO_DIRS_DEFAULT "")
 endif()
+set(TERMINFO_DIRS "${TERMINFO_DIRS_DEFAULT}" CACHE STRING "Colon-separated list of directories where unibilium should look for compiled terminfo files at runtime")
```

Comment:
Description taken from the [README](https://github.com/neovim/unibilium#building).